schema.graphql

```diff
@@ -419,6 +419,43 @@ type TransferManager @entity {
   updatedBlock: Block!
 }
 
+type StatType @entity {
+  id: ID! # ticker/op/type/claimIssuer
+  asset: Asset!
+  opType: String!
+  claimType: String
+  claimIssuer: Identity
+  createdBlock: Block!
+  updatedBlock: Block!
+}
+
+type TransferCompliance @entity {
+  id: ID! # ticker/op/type/claimIssuer
+  asset: Asset! @index(unique: false)
+  type: String!
+  statType: StatType!
+  # value is present only in case of MaxInvestorCount and MaxInvestorOwnership
+  value: BigInt
+  # following values are present in case of ClaimCount and ClaimOwnership
+  claimType: String
+  claimValue: String
+  claimIssuer: Identity
+  min: BigInt
+  max: BigInt
+  createdBlock: Block!
+  updatedBlock: Block!
+}
+
+type TransferComplianceExemption @entity {
+  id: ID! #asset/opType/claimType
+  asset: Asset! @index(unique: false)
+  opType: String!
+  claimType: String
+  exemptedEntities: [String]!
```

Comment:
I think these are usually DIDs, maybe we could make a many too many to Identities. We'd probably have to handle the case where an exemption is set for non existing Identity with an upsert like function though.  

My concern is if this gets large we'll have no pagination. To me it doesn't seem like a field this should get embedded, even though it does make the query flatter.

Might be OK to leave as it is, but its a breaking change to address later.

Reply:
Since we could add a non-existing entity, I have stored the `exemptedEntity` as string. I have removed the array part so that we could avoid the large values in this column and take advantage of pagination as we did for claims.